test_pull_requests

```diff
@@ -550,9 +698,9 @@ module Hub
       update_status(settings['test_prefix'], commit['sha'], repo, 'success', build_url, 'Passed')
       sleep 10
```

Comment:
It is a different sleep time and not a retry.  Would be fine as another constant.